cmd/osm-bootstrap/crds/config_mesh_root_certificate.yaml

```
@@ -152,6 +152,9 @@ spec:
                     - required: ["certManager"]
                     - required: ["vault"]
                     - required: ["tresor"]
+            intent:
+              description: Specifies the intent of the MeshRootCertificate can be (Active, Passive)
```

nit: Not sure if "behavior" is the best word here. What are your thoughts?
```suggestion
              description: Intent specifies the behavior of the MeshRootCertificate, can be Active or Passive
```

---

would `role` make sense here over `intent`?  for example the role of the certificate would be `active` or `passive`.

---

I can see that! I was thinking `role` was more aligned with `validating` or `issuing` rather than active or passive, but I can see either way!